src/documentation/pages/components/badge/_properties.vue

```diff
@@ -0,0 +1,94 @@
+<template lang="pug">
+  div
+    h3 Web Component
+
+    h4 Properties
```

Comment:
Let's remove the hardcoding of the properties. We need automatic generating of the properties table from docs.json. We will implement it in a separate PR.

Reply:
Perfect! :)